install-panel.sh

```diff
@@ -604,9 +604,21 @@ function debian_based_letsencrypt {
   systemctl stop nginx
 
   echo -e "\nMake sure you choose Option 1, and create a Standalone Web Server during the certificate"
-  certbot certonly -d "$FQDN"
+  certbot certonly --standalone -d "$FQDN"
```

Comment:
I like the way you check if the certificate exists or not, but this line will cause the script to exit since `set -e` is run on line 3 (which means script exists immediately if any commands return other exit codes than 0).

Reply:
I am not that keen on bash :(
I would think I need something like `return 0`?

Reply:
```suggestion
  certbot certonly --standalone -d "$FQDN" || FAILED=true
```

I believe you can do something like this. The `||` is like "else" in an "if-else"-statement. So, in this case, it will set the variable `FAILED` to true (which may then be used on line 609 to check). See the suggestion below.

Again, I am not completely sure this works in this instance. But a simple test like this can verify that it *should* work.

```bash
#!/bin/bash

set -e

failed=false
command_that_does_not_exist_and_therefore_will_fail || failed=true

echo "$failed"
```

If I run this, it prints `true` since the command does not exist and fails. If I then replace command_that_does_not_exist_and_therefore_will_fail` with a command that I know works, then the script prints `false`.